tests/petsc/update_ghosts.cc

```diff
@@ -0,0 +1,20 @@
+#include <deal.II/lac/petsc_parallel_block_vector.h>
+
+#include "../tests.h"
```

Comment:
This `#include` needs to be first in the list of includes. Please also add the standard header to the file.